ci-operator/step-registry/redhat-lp-chaos/lp/cnv/snapshot-scenario/redhat-lp-chaos-lp-cnv-snapshot-scenario-commands.sh

```
@@ -0,0 +1,169 @@
+    (restore)
+        oc wait vmrestore "${restoreName}" -n "${vmNamespace}" \
+            --for=jsonpath='{.status.complete}'=true \
+            --timeout="${waitTimeout}"
+        virtctl start "${vmName}" -n "${vmNamespace}" || :
+        ;;
+    (delete)
+        # oc wait --for=delete errors if the resource is already gone; treat not-found as success.
+        oc wait vmsnapshot "${snapshotName}" -n "${vmNamespace}" \
+            --for=delete \
+            --timeout="${waitTimeout}" 2>/dev/null || \
+            { ! oc get vmsnapshot "${snapshotName}" -n "${vmNamespace}" &>/dev/null; }
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)